src/d3a/models/market/two_sided.py

```diff
@@ -103,6 +103,9 @@ def bid(self, price: float, energy: float, buyer: str, buyer_origin: str,
         if energy <= 0:
             raise InvalidBid()
 
+        if time_slot is None:
+            time_slot = self.time_slot
```

Comment:
Great! So now all of the order objects should have a not-None `time_slot` member. 👍